CHANGELOG.md

```diff
@@ -24,7 +29,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [2.6.1] - 2021-03-17
 
 ### Fixed
-- Fix infinite loop when calling `chooseAudioInputDevice` with a `MediaDeviceInfo` instance.
+- Fix infinite loop when calling `chooseAudioInputDevice` with a
+  `MediaDeviceInfo` instance.
+- Fewer observers are now retained after meetings end. This should reduce
```

Comment:
This should be moved to the Unreleased section

Reply:
We really need to fix this whole changelog thing at some point.